models/xception/xception.py

```diff
@@ -0,0 +1,63 @@
+from tensorflow.keras.layers import Activation, Add, BatchNormalization, Conv2D, Dense, GlobalAveragePooling2D, Input, MaxPool2D, SeparableConv2D
```

Comment:
Add an initial docstring on top of the script
Use parenthesis in import statement in order to observe line length